torchci/rockset/metrics/last_successful_jobs.lambda.json

```diff
@@ -4,8 +4,8 @@
     {
       "name": "jobNames",
       "type": "string",
-      "value": "docs push / build-docs (python),docs push / build-docs (cpp)"
```

Comment:
What effect does the `, 30` and `, 180` have here?

Reply:
Well, they are the values of the timeout dimension of the job.  GHA has a way to iterate all the possible combination in the name, which makes sense.